resources/views/setup.blade.php

```diff
@@ -248,6 +248,10 @@
                 <option value='//maxcdn.bootstrapcdn.com/bootswatch/3.3.7/sandstone/bootstrap.min.css'>Sandstone</option>
                 <option value='//maxcdn.bootstrapcdn.com/bootswatch/3.3.7/cyborg/bootstrap.min.css'>Jet Black</option>
```

Comment:
Not related to your PR, but could we also remove "jet black'"? It seems to be a duplicate of "midnight black" :)

Reply:
Done.